packages/cardano-services/test/Http/HttpServer.test.ts

```diff
@@ -74,7 +75,9 @@ describe('HttpServer', () => {
   beforeEach(async () => {
     port = await getRandomPort();
     apiUrlBase = `http://localhost:${port}`;
-    provider = new DbSyncUtxoProvider(db, logger);
+    const providerInternal = new DbSyncUtxoProvider(db, logger);
```

Comment:
These test changes are out of scope, so it would be best to commit them separately.

Reply:
Rolled back with last proposal